docs/machine_learning.rst

```diff
@@ -89,30 +89,36 @@ Finally execute this whole training flow as a batch job::
     training_job = model.create_job()
     training_job.start_and_wait()
 
-
 Inference
 ----------
 
 When the batch job finishes successfully, the trained model can then be used
 with the ``predict_random_forest`` process on the raster data cube
 (or another cube with the same band structure) to classify all the pixels.
 
+Thus, the training job results is inspected to fetch the trained models url::
+
+    from openeo.rest.job import JobResults
+
+    results : JobResults = training_job.get_results()
+    links = results.get_metadata()['links']
+    ml_model_metadata_url = [link for link in links if 'ml_model_metadata.json' in link['href']][0]['href']
+    print(ml_model_metadata_url)
+
+
+Next, load the model from the URL::
+
+    from openeo.rest.mlmodel import MlModel
+    saved_model = MlModel.load_ml_model(connection=connection, id=ml_model_metadata_url)
```

Comment:
Isn't it possible to do this with something like

```python
saved_model = connection.load_ml_model(ml_model_metadata_url)
```

Reply:
yes it is indeed.